providers/common/include/prov/fipsindicator.h

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright 2023-2024 The OpenSSL Project Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License 2.0 (the "License").  You may not use
+ * this file except in compliance with the License.  You can obtain a copy
+ * in the file LICENSE in the source distribution or at
+ * https://www.openssl.org/source/license.html
+ */
+
+#ifdef FIPS_MODULE
+
+# define OSSL_FIPS_INDICATOR_ID_STRICT_CHECK0 0
+# define OSSL_FIPS_INDICATOR_ID_STRICT_CHECK1 1
+# define OSSL_FIPS_INDICATOR_ID_STRICT_CHECK2 2
```

Comment:
Are better more descriptive names possible here?  For example:

```
_GLOBAL
_CONFIG
_PARAM
```

Reply:
I have no idea what you mean here.. This is the index which will be normally 0.
I need to associate an id with a string name... these can have aliases to other names, but they still map back to "strict0", "strict1". 
We could do completely different names but the macros would have to have these names passed in multiple places...